components/proposals/20220121-notebook-idleness.md

```diff
@@ -0,0 +1,163 @@
+# Extend Notebooks Controller to expose idleness for Jupyter

```

Comment:
Let's edit the title to only include the proposed new feature. It's implied that we will need to extend the notebook controller to achieve this :-)

Here's an alternative:
```
Expose Idleness Information for Jupyter Notebooks
```

Reply:
**DONE!** I added a "review: change the title of the proposal" commit for this.